### DISCUSSION TITLE
Title customization ?

### DISCUSSION THREAD

**Discussion by nyue** [thegeeklab/hugo-geekdoc#165] [Q&A]
There is a parameter title in config.toml which I can use to customize the title, it is used in 3 locations on the front page. I am wondering if there are additional parameters associated with titles that I may use to further customize the other string so that I can have different strings

i.e. the first is `atmpypeline`, the second is `my files` and third is `my contents`

![TitleQuestions](https://user-images.githubusercontent.com/486172/125683505-de0cdc2c-d6f6-4720-be4b-11727ae35153.png)

**Comment by xoxys** [comment]
No there are no further parameters that can be used. The title in the config.toml is intended to be the title of your whole site and is only used as a fallback on 2) and 3):

![grafik](https://user-images.githubusercontent.com/3391958/125691618-df9d7c8f-219a-460d-bb6b-8cf1b4fa1633.png)

The title in 2) and 3) is set by the title of your root page, see https://github.com/thegeeklab/ansible-later/blob/main/docs/content/_index.md
while the title used in 2) cant be changed independently as it is part of the breadcrumb navigation and will therefor follow titles of the page navigation path, e.g.

![grafik](https://user-images.githubusercontent.com/3391958/125692018-5f6368dd-607c-4dda-a374-c94bfe5e9a38.png)
